docs/summary/VARIABLE_STATS.md

```diff
@@ -3140,7 +3140,7 @@ All statistics generated from the uprated (to 2022) 2019-20 Family Resources Sur
   - Non-zero count: 2,307.4921264648438
 
 
-- pension_income:
+- private_pension_income:
```

Comment:
is there public pension income?

Reply:
Technically it's `state_pension`

Reply:
So just to hammer this out - my read of these variables was that `state_pension` refers to the UK public pension program, while `private_pension_income` (formerly `pension_income`) refers to all other programs. Am I right on this?

Reply:
Yes!

Reply:
Perfect! Thanks for the clarification.